js-runtime/index.js

```diff
@@ -647,7 +1088,7 @@ module.exports = {
   },
   httplsn:  async (port) => {
     const server = http.createServer((req, res) => {
-      const connId = hashf(Math.random().toString())
+      const connId = Number(hashf(Math.random().toString()))
```

Comment:
You apparently can't use a `BigInt` as an object key, even though `BigInt` has a `toString` method. 🤨